pkg/reconciler/v1alpha1/taskrun/resources/pod_test.go

```diff
@@ -249,6 +251,52 @@ func TestMakePod(t *testing.T) {
 			},
 			Volumes: implicitVolumes,
 		},
+	}, {
+		desc: "working-dir-in-workspace-dir",
```

Comment:
I left this test in so that there's at least one test sanity checking that we add the container as expected.